### PR TITLE
🔥🔥🔥 Firebase added via re-base package 🔥🔥🔥

### PR DESCRIPTION
**Pull request by thebritican**
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

image

🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

✅ 

### After
image

### CODE REVIEW

**Comment by jden** [comment]
wow there are so many emoji happening here i don't know what's going on 


🚒 🚨

File: components/Category/Category.scss [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+  background-color: white;
+  background-image: linear-gradient(white, #ebebeb);
+  border: 1px solid #bababa;
+  border-radius: 4px;
+  font-size: 20px;
+  font-weight: 700;
+  text-align: left;
   cursor: pointer;
-  padding: 10px 0;
+  padding: 10px 0 10px 0;
```
Comment (thebritican) [inline components/Category/Category.scss]: this doesn't matter

File: components/CategoryList/CategoryList.js [hunk 2]
```diff
@@ -2,20 +2,14 @@ import React from 'react'
...
-  { name: 'icon-desktop', 'icon': 'Technology'},
-]
-
-export default () => (
+const CategoryList = (props) => (
   <ul title="Services" className="categories btn-group">
-    {categories.map(category => (
-      <li className="unselectable">
+    {props.categories.map((category, index) => (
+      <li key={index} className="unselectable">
```
Comment (craig-day) [inline components/CategoryList/CategoryList.js]: lets make this key better and do something like
```
key={`category-${index}`}
```

File: pages/index.js [hunk 3]
```diff
@@ -1,15 +1,38 @@
 import React, { Component } from 'react'
 import './root.scss'
 import CategoryList from './../components/CategoryList'
+import Rebase from 're-base'
+const base = Rebase.createClass('https://vivid-inferno-4672.firebaseio.com/')
```
Comment (craig-day) [inline pages/index.js]: pull this from `process.env`? (do we even have that?)

File: pages/index.js [hunk 4]
```diff
@@ -1,15 +1,38 @@
...
+
+const getCategories = (categoryNames) => (
+  categoryNames.map(name => (
+    { icon: 'icon-home', name: name[0].toUpperCase() + name.slice(1) }
```
Comment (craig-day) [inline pages/index.js]: Replace this with 

```javascript
String.prototype.capitalize = function() {
  this[0].toUpperCase() + this.slice(1);
}
```

then do `name.capitalize`

:trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface: :trollface:

**Comment by craig-day** [comment]
2 real comments to at least talk about, but lgtm :+1: 💥

File: pages/index.js (hunk 3 above)
Comment (jden) [inline pages/index.js]: http://stackoverflow.com/a/30061249/120713

File: pages/index.js (hunk 4 above)
Comment (jden) [inline pages/index.js]: go home @craig-day ur drunk 🍷 🍺

File: pages/index.js (hunk 3 above)
Comment (thebritican) [inline pages/index.js]: let's do this as a deploy and build PR

Comment (thebritican) [inline pages/index.js]: #281 we'll do @jden's link ty jden

File: pages/index.js (hunk 4 above)
Comment (thebritican) [inline pages/index.js]: 3 years ago everyone would be 👏 👏 👏

**Pull request merged**